iogt/static/css/modules/_m-search.scss

```diff
@@ -1,4 +1,17 @@
 
 .search {
-  text-align: center;
+  background: #f0f0f0;
+  position: relative;
+  padding: 15px;
+
+  input[type="submit"] {
```

Comment:
There are a lot of magic numbers here that's magic the code smell; check out the work I did on Tune Me that's a bit cleaner (uses flexbox): https://github.com/praekelt/molo-tuneme/blob/develop/tuneme/static/new/css/state/_s-header.scss#L6

A lot of this isn't really necessary though... because we're just waiting for the feature phone templates. Maybe just stick to the HTML for now?